extensions/eclipse-che-theia-plugin/src/che.d.ts

```diff
@@ -0,0 +1,13 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import './che-proposed';
```

Comment:
do we need this import ?

Reply:
Yes, other vice types from `che-proposed` is not visible in API implementation